src/components/toast/toast-test.stories.mdx

```diff
@@ -34,6 +34,7 @@ export const ToastStory = ({
   };
   const handleOpen = () => {
     setIsOpen(!isOpen);
+    action("open")(!isOpen);
```

Comment:
I thought it was OK to use `action` in test stories. I don't mind if we remove it, but maybe the QAs need this for some reason?

Reply:
yes, they do need it for user interaction tests